app/src/main/java/ch/epfl/sweng/SDP/MainActivity.java

```diff
@@ -17,7 +17,7 @@
 import ch.epfl.sweng.SDP.firebase.Database;
 import ch.epfl.sweng.SDP.home.HomeActivity;
 
-public class MainActivity extends Activity {
+public class MainActivity extends BaseActivity {
```

Comment:
I think we can allow the user to go back with the back button in the main activity